tests/v2/validation/rbac/rbac_etcd_backup_test.go

```diff
@@ -0,0 +1,132 @@
+package rbac
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	"github.com/rancher/rancher/tests/framework/extensions/users"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/rancher/rancher/tests/v2/validation/provisioning/rke2"
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type ETCDRbacBackupTestSuite struct {
+	suite.Suite
+	client             *rancher.Client
+	standardUser       *management.User
+	standardUserClient *rancher.Client
+	session            *session.Session
+	cluster            *management.Cluster
+	adminProject       *management.Project
+	namespace          string
+	clusterName        string
+}
+
+func (rb *ETCDRbacBackupTestSuite) TearDownSuite() {
+	rb.session.Cleanup()
+}
+
+func (rb *ETCDRbacBackupTestSuite) SetupSuite() {
+	testSession := session.NewSession()
+	rb.session = testSession
+
+	rb.namespace = defaultNamespace
+
+	client, err := rancher.NewClient("", testSession)
+	require.NoError(rb.T(), err)
+
+	rb.client = client
+	rb.clusterName = client.RancherConfig.ClusterName
+	require.NotEmptyf(rb.T(), rb.clusterName, "Cluster name to install should be set")
+	clusterID, err := clusters.GetClusterIDByName(rb.client, rb.clusterName)
+	require.NoError(rb.T(), err, "Error getting cluster ID")
+	rb.cluster, err = rb.client.Management.Cluster.ByID(clusterID)
+	require.NoError(rb.T(), err)
+}
+
+func (rb *ETCDRbacBackupTestSuite) ValidateEtcdSnapshotCluster(role string) {
+
+	/* Testcase1 Verify cluster members - Owner/member,  Project members - Owner/member are able to perform snapshot action on clusters
```

Comment:
I don't think this comment block is necessary.

Reply:
removed